src/style.css

```
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: "Avenir" ;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir" ;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.bar {
+  font-family: "Avenir", sans-serif;
+  width: 100%;
+  color: #3d4e61;
+  font-weight: 700;
```

```suggestion
  font-weight: 700;
```
This kind of font-weight should be used when the bar is in focus!